sql/core/src/test/resources/sql-tests/inputs/collations.sql

```diff
@@ -101,3 +101,319 @@ select str_to_map(text collate utf8_binary, pairDelim collate utf8_lcase, keyVal
 select str_to_map(text collate utf8_binary, pairDelim collate utf8_binary, keyValueDelim collate utf8_binary) from t4;
 
 drop table t4;
+
+-- ConcatWs
+create table t1(utf8_binary string collate utf8_binary, utf8_lcase string collate utf8_lcase) using parquet;
```

Comment:
and after we've decided on which tables we intend to keep, let's call them using different names (e.g. t1, t2, t3, etc.) instead of calling them all "t1"